templates/consul_catalog.tmpl

```diff
@@ -1,12 +1,33 @@
-[backends]{{range .Nodes}}
+[backends]
+{{range .Nodes}}
     [backends.backend-{{getBackend .}}.servers.server-{{.Node.Node | replace "." "-"}}-{{.Service.Port}}]
-    url = "http://{{.Node.Address}}:{{.Service.Port}}"
+    	url = "http://{{.Node.Address}}:{{.Service.Port}}"
+        {{$weight := getAttribute "weight" .Service.Tags}}
+        {{with $weight}}
+            weight = {{$weight}}
+        {{end}}
 {{end}}
 
+{{range .Services}}
+    {{$service := .ServiceName}}
+    {{$circuitBreaker := getAttribute "circuitbreaker" .Attributes}}
+    {{with $circuitBreaker}}
+    [backends.backend-{{$service}}.circuitbreaker]
+        expression = "{{$circuitBreaker}}"
+    {{end}}
+
+    {{$loadBalancer := getAttribute "loadbalancer" .Attributes}}
+    {{with $loadBalancer}}
+    [backends.backend-{{$service}}.loadbalancer]
+    	method = "{{$loadBalancer}}"
```

Comment:
This line contains a tab, inconsistent with the spaces used in the rest of the file.